doc/odml_ontology/root-ontology.ttl

```diff
@@ -0,0 +1,330 @@
+#    Data properties
+#################################################################
+
+###  https://g-node.org/projects/odml-rdf#author
+<https://g-node.org/projects/odml-rdf#author> rdf:type owl:DatatypeProperty ;
+                                              rdfs:domain <https://g-node.org/projects/odml-rdf#Document> ;
+                                              rdfs:range xsd:string ;
+                                              rdfs:label "author" .
+
+
+###  https://g-node.org/projects/odml-rdf#date
+<https://g-node.org/projects/odml-rdf#date> rdf:type owl:DatatypeProperty ;
+                                            rdfs:domain <https://g-node.org/projects/odml-rdf#Document> ;
+                                            rdfs:range xsd:date ;
+                                            rdfs:label "date" .
+
+
+###  https://g-node.org/projects/odml-rdf#definition
+<https://g-node.org/projects/odml-rdf#definition> rdf:type owl:DatatypeProperty ;
+                                                  rdfs:domain <https://g-node.org/projects/odml-rdf#Property> ,
+                                                              <https://g-node.org/projects/odml-rdf#Section> ;
+                                                  rdfs:range xsd:string ;
+                                                  rdfs:label "definition" .
+
+
+###  https://g-node.org/projects/odml-rdf#doc-version
+<https://g-node.org/projects/odml-rdf#doc-version> rdf:type owl:DatatypeProperty ;
+                                                   rdfs:domain <https://g-node.org/projects/odml-rdf#Document> ;
+                                                   rdfs:range xsd:float ;
+                                                   rdfs:label "doc-version" .
+
+
+###  https://g-node.org/projects/odml-rdf#dtype
+<https://g-node.org/projects/odml-rdf#dtype> rdf:type owl:DatatypeProperty ;
+                                             rdfs:domain <https://g-node.org/projects/odml-rdf#Property> ;
+                                             rdfs:range xsd:string ;
+                                             rdfs:label "dtype" .
+
+
+###  https://g-node.org/projects/odml-rdf#hasResource
+<https://g-node.org/projects/odml-rdf#hasResource> rdf:type owl:DatatypeProperty ;
+                                                   rdfs:domain <https://g-node.org/projects/odml-rdf#Terminology> ;
+                                                   rdfs:range rdf:XMLLiteral ;
+                                                   rdfs:label "hasResource" .
+
+
+###  https://g-node.org/projects/odml-rdf#hasValue
+<https://g-node.org/projects/odml-rdf#hasValue> rdf:type owl:DatatypeProperty ;
+                                                rdfs:domain <https://g-node.org/projects/odml-rdf#Property> ;
+                                                rdfs:range rdf:li ;
+                                                rdfs:label "hasValue" .
```

Comment:
I think this should be another object property to an rdf:Bag object, if we support the Bag object. If we use blank nodes, then I don't shouldn't reference rdf:li, since this belongs to rdf:Containers.